test/kino/vega_lite_test.exs

```diff
@@ -93,6 +93,21 @@ defmodule Kino.VegaLiteTest do
     assert_broadcast_event(kino, "push", %{data: [], dataset: nil, window: 0})
   end
 
+  test "configure/2" do
+    # with invalid theme
+    assert_raise ArgumentError,
+                 "invalid value for :theme, it should be one of [nil, :livebook], got: :invalid",
+                 fn -> Kino.VegaLite.configure(theme: :invalid) end
+
+    # with livebook theme
+    Kino.VegaLite.configure(theme: :livebook)
+    assert Application.get_env(:kino_vega_lite, :theme) == :livebook
+
+    # with empty theme
+    Kino.VegaLite.configure(theme: nil)
+    assert is_nil(Application.get_env(:kino_vega_lite, :theme))
```

Comment:
Instead of checking the application config, I would test if has the desired effect. Specifically, you can do `connect` as in the other tests and check if the config has `theme: nil` (that's as far as the tests go, the rest is up to the JS client).